### PR TITLE
Update doc to WiX v5

### PR DESCRIPTION
**Pull request by robmen**
Also fix up Docusaurus a bit

### CODE REVIEW

File: src/xsd4/iis.xsd [hunk 1]
```diff
@@ -692,9 +692,13 @@
           </xs:restriction>
         </xs:simpleType>
       </xs:attribute>
-      <!-- The Overwrite attribute is not implemented at this time.
-      <xs:attribute name="Overwrite" type="wxs:YesNoTypeUnion" />
-      -->
+      <xs:attribute name="Overwrite" type="wxs:YesNoTypeUnion">
+        <xs:annotation>
+          <xs:documentation>
+            [WiX v5] Allow the certificate to overwrite.
```
Comment (barnson) [inline src/xsd4/iis.xsd:698]: Still unsupported, no?

File: src/xsd4/firewall.xsd [hunk 2]
```diff
@@ -215,4 +444,135 @@
       </xs:anyAttribute>
     </xs:complexType>
   </xs:element>
+
+  <xs:simpleType name="PropertyReference">
```
Comment (barnson) [inline src/xsd4/firewall.xsd:448]: 🥇

**Review by barnson — CHANGES_REQUESTED** [review]

File: src/xsd4/iis.xsd (hunk 1 above)
Comment (robmen) [reply src/xsd4/iis.xsd:698]: I wondered about that. There is a test with it, and code was added to the compiler but I didn't see the CA change. I'll take it back out based on your comment.

Comment (barnson) [reply src/xsd4/iis.xsd:698]: We had no cert unit tests so I added one for https://github.com/wixtoolset/issues/issues/7784.

**Review by barnson — APPROVED** [review]

**Pull request merged**